Framework/API/inc/MantidAPI/DetectorInfo.h

```diff
@@ -76,18 +77,30 @@ class MANTID_API_DLL DetectorInfo {
   size_t size() const;
 
   bool isMonitor(const size_t index) const;
+  bool isMonitor(const std::pair<size_t, size_t> index) const;
```

Comment:
Passing pairs by reference now, though I am not sure if that is really better (cannot exclude that it is actually slower).

I don't think a typedef makes things better. Client code mostly uses `auto` I guess, and if it accessing the members the typedef does not help in any case.